apps/masterbots.ai/components/routes/browse/browse-list.tsx

```
@@ -72,7 +93,11 @@ export default function BrowseList() {
   }
 
   React.useEffect(() => {
```

Biome

[error] This hook specifies a dependency more specific that its captures: selectedChatbots.length

This capture is more generic than...

...this dependency.


(lint/correctness/useExhaustiveDependencies)